internal/charmstore/store.go

```diff
@@ -799,6 +799,7 @@ var seriesScore = map[string]int{
 	"yakkety": 7,
 	"zesty":   8,
 	"artful":  9,
+	"cosmic":  10,
```

Comment:
This priority should be 0 I think, no-one will be able to install these charms (although people don't really make charms for non-lts releases).